slides/workshopslides_gitDevOps.html

```diff
@@ -475,52 +411,151 @@ <h1 data-id="quarto-animate-title">What are the key blockers to you using git?</
 
 <section id="what-is-git" class="title-slide slide level1 center">
 <h1>What is git?</h1>
-<ul>
-<li>Git is a piece of software designed for version control, i.e.
+<div class="fragment fade-up">
+<p>Git is a piece of software designed for version control, i.e.</p>
+</div>
+<div class="fragment fade-up">
 <ul>
 <li>Tracking changes;</li>
+</ul>
+</div>
+<div class="fragment fade-up">
+<ul>
 <li>Undoing changes;</li>
+</ul>
+</div>
+<div class="fragment fade-up">
+<ul>
 <li>Maintaining parallel variants of code.</li>
```

Comment:
Could maybe add an extra bullet about safely storing your code somewhere central/"in the cloud"

Reply:
Have added a bit more focus on this in reference to Azure DevOps instead.